### DISCUSSION TITLE
Client IP anonymization is enabled, but they are keep logged

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#2933] [Q&A]
Hello,

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Problem Description
This is feature request. Client IP anonymization enabled, but they are keep logged in the "Clients (runtime)" section:
![image](https://user-images.githubusercontent.com/45385632/113988264-f7afc380-9857-11eb-92bb-0e8a4218dec8.png)

### Proposed Solution
Add feature to disable this function or just stop logging them when client IP anonymization is enabled. This will increase anonymity, I think🤔

**Comment by ainar-g** [comment]
Hi.  I assume that is the “Client settings” page?  Client IP anonymisation is a feature of the query log.  That is, the IPs of clients are anonymised, in the `.../data/querylog.json*` files and on the “Query Log” page.  The clients on the “Client settings” page aren't anonymised.

**Reply by ammnt** [reply]
So I suggest adding the disable or anonymize feature in this section🤔

**Reply by ameshkov** [reply]
The idea is to not allow linking IPs to queries, not in hiding the fact that AdGuard Home is used by this or that client.
